vcd/resource_vcd_org_vdc_test.go

```diff
@@ -232,3 +303,41 @@ resource "vcd_org_vdc" "{{.VdcName}}" {
   delete_recursive         = true
 }
 `
+
+const testAccCheckVcdVdc_update = `
+resource "vcd_org_vdc" "{{.VdcName}}" {
+  name = "{{.VdcName}}"
+  org  = "{{.OrgName}}"
+
+  allocation_model = "{{.AllocationModel}}"
+  network_pool_name     = "{{.NetworkPool}}"
+  provider_vdc_name     = "{{.ProviderVdc}}"
+
+  compute_capacity {
+    cpu {
+      allocated = 2048
+      limit     = 2048
+    }
+
+    memory {
+      allocated = 2048
+      limit     = 2048
+    }
+  }
+
+  storage_profile {
+    name = "{{.ProviderVdcStorageProfile}}"
+    enabled  = true
+    limit    = 10240
+    default  = true
+  }
+
+  cpu_guaranteed           = "{{.CpuGuaranteed}}"
```

Comment:
As this value is not a string anymore I guess you could remove the quotes `""`
It won't change a lot, but will be syntactically clearer (especially reviewing the built `binary-tests`.

Reply:
done